deploy/operator-local.yaml

```diff
@@ -54,3 +54,24 @@ spec:
                   fieldPath: metadata.name
             - name: OPERATOR_NAME
               value: "che-operator"
+            - name: CHE_VERSION
+              value: "7.8.0"
+            - name: IMAGE_default_che_server
+              value: "quay.io/eclipse/che-server:7.8.0"
+            - name: IMAGE_default_plugin_registry
+              value: "quay.io/eclipse/che-plugin-registry:7.8.0"
+            - name: IMAGE_default_devfile_registry
+              value: "quay.io/eclipse/che-devfile-registry:7.8.0"
+            - name: "IMAGE_default_pvc_jobs"
+              value: "registry.access.redhat.com/ubi8-minimal:8.0-213"
+            - name: "IMAGE_default_postgres"
+              value: "centos/postgresql-96-centos7:9.6"
+            - name: "IMAGE_default_keycloak"
+              value: "quay.io/eclipse/che-keycloak:7.8.0"
+            - name: "IMAGE_default_che_workspace_plugin_broker_metadata"
+              value: "quay.io/crw/pluginbroker-metadata-rhel8:2.1"
```

Comment:
Hmm... Guys, I see that our release scripts replace this images from crw to the che upstream:

`quay.io/crw/pluginbroker-metadata-rhel8:2.1 -> quay.io/eclipse/che-plugin-metadata-broker:v3.1.0`

 Maybe Should we do the same in this pr? Or Does our scripts mistake?